generators/dart/text.js

```diff
@@ -295,3 +295,53 @@ Blockly.Dart['text_prompt_ext'] = function(block) {
 };
 
 Blockly.Dart['text_prompt'] = Blockly.Dart['text_prompt_ext'];
+
+Blockly.Dart['text_count'] = function(block) {
+  var text = Blockly.Dart.valueToCode(block, 'TEXT',
+      Blockly.Dart.ORDER_UNARY_POSTFIX) || '\'\'';
+  var sub = Blockly.Dart.valueToCode(block, 'SUB',
+      Blockly.Dart.ORDER_NONE) || '\'\'';
+  // Substring count is not a native Dart function.  Define one.
+  var functionName = Blockly.Dart.provideFunction_(
+      'text_count',
+      ['int ' + Blockly.Dart.FUNCTION_NAME_PLACEHOLDER_ +
+        '(String string, String search) {',
```

Comment:
Use a more descriptive name for the first parameter--this code may be used by someone to understand how to get started in Dart.  Same for parameter names in generated functions in other languages.